### PR TITLE
[SYCL] Use teams to assign code ownership

### PR DESCRIPTION
**Pull request by vladimirlaz**

### CODE REVIEW

**Comment by vladimirlaz** [comment]
@bader, could you please review?

**Review by bader — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,86 +1,35 @@
...
+SYCL/GroupLocalMemory @Pennycook @intel/llvm-reviewers-runtime
 
-# Online compiler
-SYCL/OnlineCompiler @v-klochkov
-
-# Plugin interface
-SYCL/Plugin @smaslov-intel
+# Plugin interface for Level Zero
+SYCL/Plugin/*level-zero* @intel/dpcpp-l0-pi-reviewers
+SYCL/Plugin/*level_zero* @intel/dpcpp-l0-pi-reviewers
```
Comment (bader) [inline .github/CODEOWNERS:26]: This looks like a copy-paste bug.

Comment (vladimirlaz) [reply .github/CODEOWNERS:26]: we have tests with a minus in name and underline sign. It is expected

Comment (bader) [reply .github/CODEOWNERS:26]: What is the reason of this inconsistency?

**Pull request merged**